prow/plugins/project/project.go

```diff
@@ -147,6 +147,9 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent, p
 	if e.Action != github.GenericCommentActionCreated {
 		return nil
 	}
+	if !projectRegex.MatchString(e.Body) {
```

Comment:
I'd suggest something like: https://github.com/kubernetes/test-infra/blob/c797b2df17692eff94d80f71b19e54cb20002838/prow/plugins/pony/pony.go#L127-L131

and then pass `mat` through into `parseCommand`. That way, you only really need to pass the entire body through a regex once.

Reply:
Thanks @cblecker ! Just made the update